src/test/java/seedu/address/logic/parser/MainParserTest.java

```diff
@@ -195,19 +196,32 @@ public void parseCountryNoteCommands_unknownCommand_throwsParseException() {
     //TODO: add tests when country commands are finalized
 
     @Test
-    public void parseClientNoteCommands_addClientNote() throws Exception {
+    public void parseClientNoteCommands_addValidClientNote_parsesSuccessfully() throws Exception {
```

Comment:
Can follow the format of the rest of the file and remove the `parsesSuccessfully`

Reply:
resolved